signal_scanner_bot/transport.py

```diff
@@ -31,36 +32,76 @@ def on_status(self, status: tweepy.Status):
 
 
 ################################################################################
-# Twitter-to-signal
+# Twitter-to-Queue
 ################################################################################
-def _twitter_to_signal():
+def _twitter_to_queue():
     """
     Top level function for running the twitter-to-signal loop.
     """
     api = twitter.get_api()
     stream = tweepy.Stream(auth=api.auth, listener=Listener())
-    log.info("Stream initialized, starting to follow")
+    log.info("Twitter stream initialized, starting to follow")
     stream.filter(track=env.RECEIVE_HASHTAGS, stall_warnings=True)
 
 
-async def twitter_to_signal():
+async def twitter_to_queue():
     """
     Asynchronous interface for the twitter-to-signal loop.
     """
     loop = asyncio.get_event_loop()
     with concurrent.futures.ThreadPoolExecutor() as pool:
         try:
-            await loop.run_in_executor(pool, _twitter_to_signal)
+            await loop.run_in_executor(pool, _twitter_to_queue)
+            await asyncio.sleep(5)
```

Comment:
I _don't_ think we need a sleep here (though the current implementation may have needed one in the loop below)